src/mod_ping.erl

```diff
@@ -200,3 +215,13 @@ ping_get(Id) ->
     #xmlel{name = <<"iq">>,
            attrs = [{<<"type">>, <<"get">>}, {<<"id">>, Id}],
            children = [#xmlel{name = <<"ping">>, attrs = [{<<"xmlns">>, ?NS_PING}]}]}.
+
+-spec is_ping_error(exml:element()) -> boolean().
+is_ping_error(#xmlel{name = <<"iq">>,
+                      attrs = [{<<"type">>, <<"error">>}, _, _, _],
```

Comment:
The problem with such matching is that we require the packet to have exactly four attributes, and the type needs to be the first one. It will fail to match most of the errors sent by the client.